rffmpeg

```diff
@@ -73,11 +105,22 @@ def load_config():
     default_config_file = "/etc/rffmpeg/rffmpeg.yml"
     config_file = os.environ.get("RFFMPEG_CONFIG", default_config_file)
 
-    with open(config_file, "r") as cfgfh:
-        try:
-            o_config = yaml.load(cfgfh, Loader=yaml.SafeLoader)
-        except Exception as e:
-            fail("Failed to parse configuration file: {}".format(e))
+    if not Path(config_file).is_file():
+        log.info("No config found in {}. Using default settings.".format(config_file))
```

Comment:
I do like the idea of a stock config like this, good addition.

Reply:
I came across it when testing in toolbox, because I didn't have the config file in `/etc` and thought that it's really not needed if you want to use defaults anyways. I'll probably also add everything as env vars that take precedence over the config.